io-streams.cabal

```diff
@@ -127,7 +127,7 @@ Library
                      text               >= 0.10  && <1.3,
                      time               >= 1.2   && <1.6,
                      transformers       >= 0.2   && <0.5,
-                     vector             >= 0.7   && <0.11,
```

Comment:
Could you please also change the minor version number and add an entry to the changelog file?